system/libuv/Makefile

```diff
@@ -52,8 +52,11 @@ CFLAGS += -Wno-shadow
 CFLAGS += -DDEF_THREADPOOL_SIZE=CONFIG_LIBUV_THREADPOOL_SIZE
 CFLAGS += -DDEF_THREADPOOL_STACKSIZE=CONFIG_LIBUV_THREAD_STACKSIZE
 
-GCCVER = $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+\.[0-9]+).*/\1/')
-ifeq ($(GCCVER),12.2.1)
+ifeq ($(GCCVER),)
+  GCCVER = $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+).*/\1/' | cut -d'.' -f1)
```

Comment:
do we need
```suggestion
  export GCCVER = $(shell $(CC) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+).*/\1/' | cut -d'.' -f1)
```
?

Reply:
let's merge this patch to unblock ci. @anchao please create a new patch if @pkarashchenko 's suggestion looks good.

Reply:
Fix https://github.com/apache/nuttx-apps/pull/1991